lib/avromatic/model/attribute_type/union.rb

```diff
@@ -0,0 +1,25 @@
+module Avromatic
+  module Model
+    module AttributeType
+      class Union
```

Comment:
Probably worth a few class level comments on how this integrates into the Virtus framework.